app/src/main/java/com/github/se/polyfit/ui/components/card/TitleAndTogleCard.kt

```diff
@@ -0,0 +1,85 @@
+package com.github.se.polyfit.ui.components.card
+
+import android.util.Log
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.Spacer
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.layout.width
+import androidx.compose.material3.ButtonDefaults
+import androidx.compose.material3.Card
+import androidx.compose.material3.CardDefaults
+import androidx.compose.material3.FilledTonalButton
+import androidx.compose.material3.MaterialTheme
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.getValue
+import androidx.compose.runtime.livedata.observeAsState
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.platform.testTag
+import androidx.compose.ui.text.font.FontWeight
+import androidx.compose.ui.unit.dp
+import com.github.se.polyfit.viewmodel.recipe.RecipeRecommendationViewModel
+
+@Composable
+fun TitleAndToggleCard(
+    title: String,
+    button1Title: String,
+    button2Title: String,
+    recipeRecViewModel: RecipeRecommendationViewModel,
+) {
+  val showIngredient by recipeRecViewModel.showIngredient.observeAsState(initial = true)
```

Comment:
### Question 
-  Have you considered making the `initial` value of `showIngredient` a parameter of the `TitleAndToggleCard` composable? This would increase flexibility and allow different initial states depending on usage context.

Reply:
That's a great idea. Thanks for the suggestion !